tests/rados/test_osd_replicated_inconsistency_scenario.py

```diff
@@ -97,12 +98,26 @@ def run(ceph_cluster, **kw):
         auto_repair_value = mon_obj.get_config(
             section="osd", param="osd_scrub_auto_repair"
         )
-        log.info(f"Original value of osd_scrub_auto_repair is {auto_repair_value}")
+        msg_auto_repair = (
+            f"Original value of osd_scrub_auto_repair is {auto_repair_value}"
+        )
+        log.info(msg_auto_repair)
         if auto_repair_value is True:
             log.error("The default value of the osd_scrub_auto_repair should be false")
             return 1
 
+        log.info("====VERIFICATION OF THE TESTS BY PERFORMING SCRUB OPERATIONS====")
```

Comment:
Let's add a check here, making sure that there are no scrubs/ deep scrubs already in progress for any of the test pools here.

Reply:
Can be added as part of next PR. required for both replicated and EC pools.

Reply:
Jira task created - https://issues.redhat.com/browse/RHCEPHQE-19942